src/predefined-variables.ts

```diff
@@ -13,16 +13,27 @@ export function init ({gitData, argv, envMatchedVariables}: PredefinedVariablesO
     // 1. gitlab variables files
     // 2. values derieved implicitly from `git remote -v`
     // 3. default value
+    const CI_SERVER_PROTOCOL = envMatchedVariables["CI_SERVER_PROTOCOL"]
```

Comment:
It's 2024, i wonder who still uses http HAHA

anyways, this is still nice to have

Reply:
I know, right? Even in my internal instance, HTTPS is used. That said, HTTP was previously used when the server was completely isolated from the Internet. So as long as GitLab continues to support it, so should we.